src/main/java/cubicchunks/server/PlayerCubeMapEntry.java

```diff
@@ -147,7 +147,7 @@ public boolean sendToPlayers() {
 		if (this.sentToPlayers) {
 			return true;
 		}
-		if (this.cube == null || !this.cube.getGeneratorStage().isLastStage()) {
+		if (this.cube == null || !this.playerCubeMap.getWorld().getGeneratorPipeline().isLastStage(cube.getGeneratorStage())) {
```

Comment:
Why even have the global LIVE stage? Why not have it implictly registered, with final isFinalStage method?